tests/codegen/snapshots/python/mutation_with_object.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+
+class addBlogPostsResultAddBlogPostsPosts:
```

Comment:
should this be `AddBlogPostsResultAddBlogPostsPosts`

Reply:
From what I can tell, this comes from the name of the mutation.  So I can change it `mutation addBlogPosts` -> `mutation AddBlogPosts`.

In my custom plugin, I capitalize the first letter before I write it in all cases.  In any case, we may want to consider that as a future enhancement to the python plugin.

I can update the test resource in this one to get UpperCamelCase.  That's easy-peasy.